### PR TITLE
Freemod/1.13.4

### PR DESCRIPTION
**Pull request by Goobwabber**
Freemod should work, I'm not sure yet
note to self: remove commented player count code before merge
issue with player spot distance not calculating (at all?)

### CODE REVIEW

File: MultiplayerExtensions/HarmonyPatches/CustomSongsPatches.cs [hunk 1]
```diff
@@ -105,4 +105,21 @@ static void Postfix(ref GameplayModifierToggle[] ____gameplayModifierToggles)
             ____gameplayModifierToggles.ToList().Find(toggle => toggle.gameplayModifier.modifierNameLocalizationKey == "MODIFIER_SLOWER_SONG").toggle.interactable = speedModsEnabled;
         }
     }
+
+    [HarmonyPatch(typeof(LobbyGameStateController), nameof(LobbyGameStateController.startedBeatmapId), MethodType.Setter)]
+    internal class AprilFoolsPatch
```
Comment (harry1453) [inline MultiplayerExtensions/HarmonyPatches/CustomSongsPatches.cs:110]: uhhhhhh XD

**Comment by harry1453** [comment]
Will this fix #103?

**Comment by Ectalite** [comment]
It does, I compiled the dev branch and it does work !

**Comment by Goobwabber** [comment]
Sorry to anyone waiting for an update, real life happened and I'm not sure when I can get back to this. For now, you should be able to use this version until It's polished and ready: https://github.com/Zingabopp/MultiplayerExtensions/actions/runs/648247814

**Pull request merged**